src/krasa/grepconsole/gui/MainSettingsForm.form

```diff
@@ -9,20 +9,20 @@
       <minimumSize width="750" height="200"/>
       <opaque value="false"/>
     </properties>
-    <border type="none"/>
+    <border type="empty"/>
```

Comment:
This felt like it made the, already busy, setting menu feel a bit cleaner, but that's just my opinion. Am open to putting it back to the way it was.